javascript/core/src/verbs/merge.ts

```diff
@@ -26,13 +26,33 @@ export interface MergeArgs extends InputColumnListArgs, OutputColumnArgs {
 	 * If it is not supplied, the values are just mashed together.
 	 */
 	delimiter?: string
+	unhot?: boolean
+	prefix?: string
+	keepOriginalColumns?: boolean
 }
 
 export const mergeStep: ColumnTableStep<MergeArgs> = (
 	input,
-	{ columns = [], strategy, to, delimiter = '' },
+	{
+		columns = [],
+		strategy,
+		to,
+		delimiter = '',
+		unhot = false,
+		prefix = '',
+		keepOriginalColumns = false,
+	},
 ) => {
-	const isSameDataTypeFlag: boolean = isSameDataType(input, columns)
+	const copyTable: ColumnTable = fromJSON(input.toJSON())
+
+	if (unhot) {
+		unhotOperation(copyTable, columns, prefix)
+	}
+
+	const isSameDataTypeFlag: boolean = isSameDataType(
+		unhot ? copyTable : input,
```

Comment:
You could get rid of all of these `unhot ? copyTable : input` if you create a single intermediate table earlier and just use it from there on out. For example:

`const working = unhot ? unhotOperation(input...) : input`

Then just use `working` directly afterwards instead of the ternary operator